VisionDemo/VisionDemo/Modules/Photo/ViewModel/PhotoViewModel.swift

```diff
@@ -0,0 +1,173 @@
+//
+//  PhotoViewModel.swift
+//  VisionDemo
+//
+//  Created by Dory on 25/11/2021.
+//
+
+import Combine
+import Foundation
+import RealmSwift
+
+final class PhotoViewModel {
+	enum Purpose {
```

Comment:
Feels like adding this enum and checking what's the Purpose in each method just made things more complicated. Perhaps would be better to have 2 view models?